src/EnergyPlus/Autosizing/HeatingAirflowUASizing.hh

```diff
@@ -71,7 +71,7 @@ namespace EnergyPlus {
         Array1D<EnergyPlus::DataSizing::TermUnitSizingData> termUnitSizing;
         Array1D<EnergyPlus::DataSizing::ZoneSizingData> finalZoneSizing;
         Array1D<EnergyPlus::DataSizing::SystemSizingData> finalSysSizing;
-        Array1D<EnergyPlus::DataSizing::ZoneEqSizingData> zoneEqSizing;
+//        Array1D<EnergyPlus::DataSizing::ZoneEqSizingData> zoneEqSizing;
```

Comment:
Change to get unit tests to run. Why is this array declared here and in the base class? In a hurry I suspect?